### DISCUSSION TITLE
API call pagination limit

### DISCUSSION THREAD

**Discussion by harinij** [outline/outline#1558] [General] [answered]
I am currently trying to see who has contributed more in our org to create a leaderboard of sorts, to allow for healthy competition and more engagement/contribution for knowledge sharing.

I am using the API call https://app.getoutline.com/api/events.list with the following payload and hitting a max pagination limit of 100. 
`{
"offset": 0,
"sort": "updatedAt",
"direction": "DESC",
"auditLog": true
}`

Is there any way to get the entire history or atleast last one month worth of data using this API. Would be great to hear your thoughts or comments. Thanks!

**Comment by tommoor** [comment, marked as answer]
Hey @harinij – seems like a reasonable approach, you just need to request multiple pages of results – you can increment the `offset` by the page size and loop requests until you hit the date you're interested in
